notifications/locale/ar/LC_MESSAGES/django.po

```diff
@@ -8,201 +8,199 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-06-24 07:55+0000\n"
+"POT-Creation-Date: 2020-11-30 13:42+0000\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=6; plural=n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5;\n"
+"Plural-Forms: nplurals=6; plural=n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 "
+"&& n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5;\n"
 
-#: notifications/apps.py:6
+#: apps.py:6
 msgid "notifications"
 msgstr "إشعارات ؛ إخطارات ؛ تبليغات"
 
```

Comment:
@batpad This looks little weird `notifications/apps.py` -> `apps.py`

Did you run from project root? Could you try that?
Also, running it once from project root should be enough instead of doing it from each app folder.

Reply:
> Also, running it once from project root should be enough instead of doing it from each app folder.

@thenav56 I get this error when trying to run it from project root:

```
$ python3 manage.py makemessages -l en -l es -l ar -l fr
Warning: No elasticsearch host found, will not index elasticsearch
2020-12-01 09:57:47 WARNING  No Azure credentials found, falling back to local logs.
[2020-12-01 09:57:47,209] logger: WARNING - No Azure credentials found, falling back to local logs.
CommandError: Unable to find a locale path to store translations for file manage.py

```

Reply:
@batpad You have to create a directory `locale` at the root.

Due to the use of `ugettext` here https://github.com/IFRCGo/go-api/blob/develop/middlewares/middlewares.py#L63, Django is looking for `locale` directory at the root level.